l2-contracts/contracts/bridge/L2SharedBridge.sol

```diff
@@ -31,9 +37,6 @@ contract L2SharedBridge is IL2SharedBridge, IL2SharedBridgeLegacy, Initializable
     /// @dev Bytecode hash of the proxy for tokens deployed by the bridge.
     bytes32 internal l2TokenProxyBytecodeHash;
 
-    /// @dev A mapping l2 token address => l1 token address
-    mapping(address l2TokenAddress => address l1TokenAddress) public override l1TokenAddress;
```

Comment:
I was not sure that we need it per se

Reply:
So here I thought we would be upgrading the current L2SharedBridge, which itself is an upgrade of the L1ERC20Bridge, so we cannot change the storage layout, meaning Initializable has to stay, and l1TokenAddress also has to stay ( we can deprecate it if we want to)

Reply:
Ah okay got it